scripts/src/models/orders.ts

```diff
@@ -214,7 +214,7 @@ export const Order = {
 		const query = OrderImpl.createQueryBuilder("ordr")
 			.innerJoinAndSelect("ordr.contexts", "context")
 			.leftJoinAndSelect("context.user", "user")
-			.orderBy("ordr.current_status_date", "DESC")
+			.orderBy("ordr.currentStatusDate", "DESC")
```

Comment:
This is weird. It didn't kept failing without changing this but this code is in production too